specs/derivation.md

```diff
@@ -437,16 +422,15 @@ Our architecture decomposes the derivation process into a pipeline made up of th
 
```

Comment:
nit: prefer numbered lists always using `1.` and then the markdown renderer will auto increment them, this reduces diff when changing orders or adding new items

Reply:
Agreed, but I've been told not to do this by someone else working on specs before, can't remember who. Shall I just change it to `1.` everywhere in a separate PR, so we never have to touch the numbering again?

Reply:
Sure, sorry for the overhead